### DISCUSSION TITLE
JamfUploader pulling old SMB_PASSWORD from somewhere

### DISCUSSION THREAD

**Discussion by NoahDowdDraper** [grahampugh/jamf-upload#159] [Q&A] [answered]
I'm having an issue with JamfUploader I believe, where it's attempting to use an old password for the smb user, which inevitably results in the account being locked out.
The password is up to date in my ~/Library/Preferences/com.github.autopkg.plist but when I run autopkg verbosely, it's definitely trying to use an old password according to the JamfPackageUploader 'Input' line. Is there anywhere else I need to change this password?

Thanks!

**Comment by grahampugh** [comment, marked as answer]
I'd need to see a full verbose output to have any clue about your setup.

**Reply by NoahDowdDraper** [reply]
Ah it's ok. I uninstalled/reinstalled JamfUploader and it's been working fine since.
